[[Documentation]] printerColon - COMn: references

seaniepie opened the pull request:
- Added an alias to the COM1: and COM9: keyword display in the description
- Replaced COM1: with COMncolon to link to the keyword dictionary item correctly
- Add links to serialControlString property

seaniepie committed:
[[Documentation]] printerColon - COMn: references

- Added an alias to the COM1: and COM9: keyword display in the description
- Replaced COM1: with COMncolon to link to the keyword dictionary item correctly
- Add links to serialControlString property

livecodeali commented on docs/dictionary/keyword/printercolon.lcdoc:
```diff
@@ -21,24 +21,24 @@ Use the <printer:> <keyword> to communicate through the printer serial
-command (glossary), Mac OS (glossary), port (glossary), COM1: (keyword),
-COM9: (keyword)
+command (glossary), Mac OS (glossary), port (glossary), 
+<COMnColon|COMn:> (keyword), serialControlString (property)
```
This should just be `COMn: (keyword)` - the references don't understand aliases like that.

seaniepie replied:
Very good. I wasn't entirely sure that it would understand the colon itself.

seaniepie committed:
[[Documentation]] COMn: - replace colon with :

Removed reference to COMnColon and used COMn: instead

livecodeali commented on docs/dictionary/keyword/printercolon.lcdoc:
```diff
@@ -31,14 +31,14 @@ To write data to the printer port, use the <write to file> <command>.
 property (glossary), keyword (glossary), Windows (glossary),
 command (glossary), Mac OS (glossary), port (glossary), 
-<COMnColon|COMn:> (keyword), serialControlString (property)
+COMn: (keyword), serialControlString (property)
```
As it turns out, your concerns were justified! Having something with a colon after it at the beginning of a line breaks the entry. It seems to work if you swap the order or `COMn: (keyword), serialControlString (property)` around to `serialControlString (property), COMn: (keyword)` though.

seaniepie replied:
That makes sense. Travis is having problems with first word with colon entries too.

seaniepie committed:
Update - Swap last two References

COMn: causes a number of issues when at the start of a line so it's been swapped to later in the line

seaniepie commented:
That should be it

livecodeali commented:
@livecode-vulcan review ok e74cdb7

livecode-vulcan commented:
💙 review by @livecodeali ok e74cdb7c9ccb711d679bb1f46671b9dc2c8fa8ae

livecode-vulcan commented:
😎 test success e74cdb7c9ccb711d679bb1f46671b9dc2c8fa8ae
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/390)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/395)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/390)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/390)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/396)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/390)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/390)
